docs/aubuf/config

```diff
@@ -0,0 +1,56 @@
+#audio_path		/usr/local/share/baresip
+audio_player		pulse,
```

Comment:
libre and baresip are 2 different projects.

Baresip knows about Libre, but libre does not know about any application using it (e.g. baresip).

Please dont put references to `baresip` inside the libre project.